helm/cloud-provider-azure/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+description: A Helm chart for installing kubernetes-sigs/cloud-provider-azure components
+home: https://raw.githubusercontent.com/kubernetes-sigs/cloud-provider-azure/master/helm/cloud-provider-azure/README.md
+keywords:
+- cloud-provider-azure
+- kubernetes-sigs
+- kubernetes
+maintainers:
+- email: jackfrancis@gmail.com
```

Comment:
maintainers should probably match https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/OWNERS#L1

Reply:
@feiskyer @andyzhangx what do you think? I would be happy to maintain this going forward (updating with every release). Perhaps @lzhecheng can be a co-maintainer?

Also happy to add everyone who is a repo maintainer.

Reply:
Please add me to the list. Thank you.

Reply:
yes, thanks for joining as co-maintainer